tests/test_plugin.py

```diff
@@ -214,5 +214,55 @@ def test_it_adapts_splits_based_on_new_and_deleted_tests(
         ]
 
 
+class TestHasExpectedOutput:
+    def test_prints_splitting_summary(self, example_suite, capsys):
+        result = example_suite.inline_run("--splits", "1", "--group", 1)
+        assert result.ret == 0
+
+        outerr = capsys.readouterr()
+        assert "[pytest-split] Running group 1/1 (10/10) tests" in outerr.out
+
+    def test_prints_splitting_summary_when_group_missing(self, example_suite, capsys):
+        result = example_suite.inline_run("--splits", "1")
+        assert result.ret == 0
+
+        outerr = capsys.readouterr()
+        assert "[pytest-split] Not splitting tests because the `group` argument is missing" in outerr.out
+
+    def test_prints_splitting_summary_when_splits_missing(self, example_suite, capsys):
+        result = example_suite.inline_run("--group", "1")
+        assert result.ret == 0
+
+        outerr = capsys.readouterr()
+        assert "[pytest-split] Not splitting tests because the `splits` argument is missing" in outerr.out
+
+    def test_prints_splitting_summary_when_splits_missing(self, example_suite, capsys):
+        result = example_suite.inline_run("--splits", 1, "--group", "1")
+        assert result.ret == 0
+
+        outerr = capsys.readouterr()
+        assert "[pytest-split] Not splitting tests because the durations_report is missing" in outerr.out
+
+    def test_prints_splitting_summary_when_durations_missing(self, example_suite, capsys, durations_path):
+        test_name = "test_prints_splitting_summary_when_durations_missing"
+        with open(durations_path, "w") as f:
+            json.dump([[f"{test_name}0/{test_name}.py::test_1", 0.5]], f)
+
+        result = example_suite.inline_run(
+            "--splits", 1, "--group", "1", "--durations-path", durations_path, "--store-durations"
+        )
+        assert result.ret == 0
+
+        outerr = capsys.readouterr()
+        assert "[pytest-split] Not splitting tests because we are storing durations" in outerr.out
+
+    def test_prints_splitting_summary_when_no_pytest_split_arguments(self, example_suite, capsys):
+        result = example_suite.inline_run()
+        assert result.ret == 0
+
+        outerr = capsys.readouterr()
+        assert "[pytest-split] Running group 1/1 (10/10) tests" in outerr.out
```

Comment:
I think it'd be best to not mention anything about pytest-split if the invocation doesn't contain any pytest-split related args

Reply:
Ok, I will change this